src/content/tools/devtools/deep-links.md

```diff
@@ -26,8 +24,8 @@ and import a Flutter project that contains deep links.
 ![Screenshot of the Deep Link Validator](/assets/images/docs/tools/devtools/deep-link-validator.png){:width="100%"}
 
 This tool helps you identify and troubleshoot any errors
-in your Android deep link setup,
-from website configuration to Android manifest files.
+in your mobile deep link setup,
+from website configuration to manifest files.
```

Comment:
manifest files is for Android, iOS use info.plist
I suggest to say "from website configuration to app settings like manifest files and info.plist" here